### ISSUE TITLE
The calculation formula of PSNR

### ISSUE THREAD

**Issue by CShiru** [jzyustc/MBRS#4]
Why is the value of `mav_val` not 255 but 2.
```
psnr = kornia.losses.psnr_loss(encoded_images.detach(), images, 2)
```
Acctually, since the channel obtained using DataLoader is RGB, with a value range of 0-1, the calculation formula of PSNR should be
```
psnr = kornia.losses.psnr_loss(encoded_images.detach()*255, images*255, 255)
```
This error causes the calculation result of PSNR to be larger than the correct one.

**Comment by CShiru** [comment]
Or it can be written like this
```
psnr = kornia.losses.psnr_loss(encoded_images.detach(), images, 1)
```

**Comment by jzyustc** [comment]
The value range is -1 to 1 due to the normalization in the dataloader, 

https://github.com/jzyustc/MBRS/blob/c0f31767bfe4262ea633c5779dacda6264795f7a/utils/Dataloader.py#L23

so I set `max_val` to 2 to match the value range.

**Comment by CShiru** [comment]
OK, thank you for your answer.

**Comment by GoXian** [comment]
> The value range is -1 to 1 due to the normalization in the dataloader,
> 
> https://github.com/jzyustc/MBRS/blob/c0f31767bfe4262ea633c5779dacda6264795f7a/utils/Dataloader.py#L23
> 
> so I set `max_val` to 2 to match the value range.

input_image range is [-1, 1]，and max_val: The maximum value in the input tensor. so I think it is
psnr = kornia.losses.psnr_loss(encoded_images.detach(), images, 1)
